lib/jekyll/configuration.rb

```diff
@@ -11,7 +11,8 @@ class Configuration < Hash
       'destination'   => File.join(Dir.pwd, '_site'),
       'plugins'       => '_plugins',
       'layouts'       => '_layouts',
-      'data_source'   =>  '_data',
+      'data_source'   => '_data',
+      'includes_dir'  => '_includes',
```

Comment:
Ah, yes, that might be confusing. For consistency, it would be nice to have `plugins_dir`, `layouts_dir`, and so forth as well. But We'd need the backwards-compatibility layer (in Jekyll::Configuration so should be easy). Want to do that? Would be great to have :)